apps/07jquery-tic-tac-toe/script.js

```diff
@@ -2,4 +2,177 @@
 
 $(document).on('ready', function() {
     // Put app logic in here
+
+    var playerTurn = 'X';
+    var numberOfTurns = 0;
+    var isWinner = false;
+
+    $("[data-cell]").click(function() {
+
+
+        if (isWinner === false) {
+
+            if ($(this).text() === '') {
+
+                $('#error').text('');
+
+                $(this).text(playerTurn);
+
+                var cellContents0 = $("[data-cell='0']").text();
+                var cellContents1 = $("[data-cell='1']").text();
+                var cellContents2 = $("[data-cell='2']").text();
+                var cellContents3 = $("[data-cell='3']").text();
+                var cellContents4 = $("[data-cell='4']").text();
+                var cellContents5 = $("[data-cell='5']").text();
+                var cellContents6 = $("[data-cell='6']").text();
+                var cellContents7 = $("[data-cell='7']").text();
+                var cellContents8 = $("[data-cell='8']").text();
+
+                checkForWin(cellContents0, cellContents1, cellContents2, cellContents3, cellContents4, cellContents5, cellContents6, cellContents7, cellContents8);
```

Comment:
We talked about this in class, I know you're already hard at work improving it :D